pkg/scheduler/util/utils.go

```diff
@@ -133,6 +133,16 @@ func PatchPod(cs kubernetes.Interface, old *v1.Pod, new *v1.Pod) error {
 	if err != nil {
 		return fmt.Errorf("failed to create merge patch for pod %q/%q: %v", old.Namespace, old.Name, err)
 	}
+
+	modifiedMap := map[string]interface{}{}
+	err = json.Unmarshal(patchBytes, &modifiedMap)
+	if err != nil {
+		fmt.Println("unable to unmarshal patchBytes:", err)
+	}
+	if n := len(modifiedMap); n == 0 {
+		return nil
+	}
+
```

Comment:
This logic feels like overkill to me. I'd expect we add this (or similar) logic only when necessary, or at least this behavior can be verified by a test case.

Reply:
yes, follow @alculquicondor 's suggestion, the change should be light now.